### DISCUSSION TITLE
Where to add additional css or scss?

### DISCUSSION THREAD

**Discussion by rbucks** [pelican-plugins/tailwindcss#6] [Q&A] [answered]
Sorry noob question here. Where should we add our own css or scss? Can this go directly in `input.css`?

**Comment by lcfd** [comment, marked as answer]
Hello! 👋 

Exactly, in order to create your own style you have to use `input.css`.

I suggest you give a look at https://tailwindcss.com/docs/adding-custom-styles#using-css-and-layer

For other such questions, I ask you to use Discussions 😄 Thank you!
